requirements.txt

```diff
@@ -0,0 +1,6 @@
+openai
```

Comment:
in your requirements it would even be better to write the packages' versions needed.
sometimes, you may install the most recent version, but some methods would be deprecated.
those methods may be methods you used.